### ISSUE TITLE
 Omxplayer is required for audio playback

### ISSUE THREAD

**Issue by Ryan-M-Smith** [Ryan-M-Smith/Quinton-VoiceAssistant#30]
# Problem

When I first started on Quinton-VoiceAssistant, I was developing on Raspberry Pi OS intending to only run it on Raspberry Pis in my house. However, I realized a few months ago that I really wanted to share my project so I posted it on GitHub. A few days ago, I upgraded to a Pi 4 and am now running Manjaro 64-bit (to take advantage of the 64-bit architecture). When I tried to run the software, I got errors because it couldn't access `omxplayer`; **a Raspberry Pi exclusive program**. Because I hadn't originally planned to put the code in a position where it would be running on other systems, it was never developed to use a universally available audio player.

**TL;DR: If you don't have a Raspberry Pi running Raspberry Pi OS with** `omxplayer` **installed, you probably can't run the software.**

# Solution

To fix this, I'll be adding in support for a universal audio player on both Linux and macOS; `ffmpeg`. As an added bonus, I'll be keeping the `omxplayer` support for any users running this on a Raspberry Pi (at least for now).

---

Sorry for the inconvenience, and keep an eye out for the bugfix; v0.3.1!
